transcription/transcription.go

```diff
@@ -29,3 +30,15 @@ func msgHeaders(from string, to []string, subject string) string {
 	msgHeaders := []string{fromHeader, toHeader, subjectHeader}
 	return strings.Join(msgHeaders, "\r\n")
 }
+
+// ConvertAudioIntoRequiredFormat converts encoded audio into the required format.
+func ConvertAudioIntoRequiredFormat(fn string) error {
```

Comment:
Side-note: "static" functions are just called functions in Go and "dynamic" functions (on a struct) are called methods.

I see what you are saying. The difference is that with email, both functions were static while now `Run` is a method on type `Cmd`.

The [`withmock` docs](https://godoc.org/github.com/qur/withmock) detail how to mock methods.

> Once you have set the controller then you can set your mock expectations, either using the EXPECT() function for function expectations, or the EXPECT() method for any method expectations. For example, if there was a type called UsefulType, and we were expecting it's HandyMethod to be called - followed by a message printed indicating the result, we might set our expectations as follows:

>```
// Create a UsefulType instance
ut := &ext.UsefulType{}

>// We expect to see HandyMethod called, and we want to return true
ut.EXPECT().HandyMethod().Return(true)

>// And we expect to see our returned value printed
mockfmt.EXPECT().Printf("HandyMethod returned: %v\n", true)
```

You will want to mock a `exec.Cmd` instead of a `ext.UsefulType`. Make sense?